### ISSUE TITLE
Add hidden text to intranet header logo

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#2091]
Currently, the intranet header logo is not accessible because it has not text content: 

![Image](https://github.com/swisspost/design-system/assets/33580481/9ef78d2b-9426-4b5b-b709-a07d57d4f9ac)
